wrapanapi/systems/redfish.py

```diff
@@ -4,6 +4,8 @@
 """
 from __future__ import absolute_import
 
+import redfish_client
```

Comment:
Sorry, only realized now: there should be no empty line between this one and `from __funture__ ...` above according to guidelines https://cfme-tests.readthedocs.io/en/master/guides/dev_guide.html#code-style

Reply:
I think that in this case the space should be OK, because `redfish_client` is in a different group than `from _future__`.

Reply:
I think the space here should be ok because `redfish_client` belongs in a different group of imports.